integration-tests/test.sh

```diff
@@ -0,0 +1,206 @@
+#! /bin/bash
```

Comment:
Shall we remove the existing files and update the job config?

https://github.com/wso2/testgrid-job-configs/blob/master/wso2is-intg-testgrid-yaml#L41

Reply:
Changes apply. Please review 

[](https://github.com/wso2/testgrid-job-configs/pull/144)
[](https://github.com/wso2/testgrid-job-configs/pull/145)